.registry/resources/minimalaws.ui-schema.yaml

```diff
@@ -1,47 +1,58 @@
-version: v0.2.0
+version: v0.2.1
```

Comment:
looks like indentation is off in this file so it might throw that parsing error like we ran into in the past